cmd/chartmuseum/main.go

```diff
@@ -116,6 +117,7 @@ func cliHandler(c *cli.Context) {
 		Host:                   conf.GetString("listen.host"),
 		PerChartLimit:          conf.GetInt("per-chart-limit"),
 		WebTemplatePath:        conf.GetString("web-template-path"),
+		ArtifactHubRepoID:      conf.GetStringMapString("artifact-hub-repo-id"),
```

Comment:
will it be nil if no flag set ? Will be panic if no `artifact-hub-repo-id` flag set but still visit the `/artifact-repo.yaml` ?